### ISSUE TITLE
Can the model reach the loss value and accuracy given in the paper?

### ISSUE THREAD

**Issue by Adenialzz** [idealo/image-quality-assessment#94]
Can the model reach the values given in the paper with backbone network is MobileNet, VGG16, Inception-V2 respectively. And according to my experiments, It can not. So I wonder if there something wrong in my implementation. It will be appreciated if you can share your experiment result with me.

**Comment by Asomey** [comment]
hello bro, i want to ask a question that how can l evaluate the network‘s performance. Since i saw many evaluation standards given by authors like EMD,LCC,SRCC.  How can i calculate these evaluation standards. Thanks bro.

**Comment by Adenialzz** [comment]
> 
hi, bro. The scipy library already provides a method to calculate the correlation coefficient:
from scipy.stats import pearsonr
from scipy.stats import spearmanr
As for emd, you can check the src/utils/losses.py file in this repo.

**Comment by gohna** [comment]
Hello, I would like to ask whether there is a big difference between your Mobilenet and VGG, because I need pre-trained NimA for other tasks, but the result I directly predicted by using this pre-trained model is quite different from that in the paper, I want to know whether it is the problem of Backbone or the startup parameters of Nima
